collector/hadoop/README.md

```diff
@@ -0,0 +1,118 @@
+
+---
+# Ingest metrics using the Hadoop integration
+
+The OTEL Collector has a variety of [third party receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver) that provide integration with a wide variety of metric sources.
+
+Please note that not all metrics receivers available for the OpenTelemetry Collector have been tested by Lightstep Observability, and there may be bugs or unexpected issues in using these contributed receivers with Lightstep Observability metrics. File any issues with the appropriate OpenTelemetry community.
+{: .callout}
+
+## Requirements
+
+* OpenTelemetry Collector Contrib v0.53.0+
+
+## Prerequisites
+
+You must have a Lightstep Observability [access token](/docs/create-and-manage-access-tokens) for the project to report metrics to.
+
+## Running the Example
+
+You can run this example with `docker-compose up` in this directory. Using `docker-compose --profile loadgen up` also creates an instance to send requests to the Hadoop service. You'll want to view this in Lightstep with a dashboard. 
+
+#### Changes
+
+Version 2.0.0 introduces uses wait_for_it script for the cluster startup
+
+#### Hadoop Docker
+
+### Quick Start
+
+To deploy an example HDFS cluster, run:
+``` sh
+  docker-compose up
+```
+
+Run example wordcount job:
+``` sh
+  make wordcount
+```
+
+Or deploy in swarm:
+``` sh
+docker stack deploy -c docker-compose-v3.yml hadoop
+```
+
+`docker-compose` creates a docker network that can be found by running `docker network list`, e.g. `dockerhadoop_default`.
```

Comment:
When I run this command I find a network called `hadoop_integrations`.

Reply:
Corrected network.